eeraster.py

```diff
@@ -173,9 +173,9 @@ def get_props(ee_asset_path: str):
   isoscape_metadata = ee.data.getAsset(ee_asset_path)
   return isoscape_metadata['properties']
 
-def del_prop(ee_asset_path: str, property_name: str):
+def delete_prop(ee_asset_path: str, property_name: str):
   """
-  del_prop function
+  delete_prop function
```

Comment:
Are we using this method somewhere where the reference (given the name change) should be changed?

Reply:
That was the original intent when this was meant to replace local isoscape stamping yes, but that is no longer true. We're stamping isoscapes locally and in EE.